Elevel/Elevel.Infrastructure.Services/Implementation/FileService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Elevel.Application.Interfaces;
+using Elevel.Domain.Models;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Elevel.Infrastructure.Services.Implementation
+{
+    public class FileService : IFileService
+    {
+        private readonly IWebHostEnvironment _hostingEnvironment;
+
+        public FileService(IWebHostEnvironment hostingEnvironment)
+        {
+            _hostingEnvironment = hostingEnvironment;
+        }
+
+        public void UploadFiles(List<IFormFile> files)
+        {
+            const string target = @"wwwroot\files";
```

Comment:
it's bad practice to have hadrdcoded separator. Pleae use Path.Combine("wwwroot", "files") + create readonly field to avoid path duplication